examples/virtual_terminal/aux_inputs/main.cpp

```diff
@@ -0,0 +1,199 @@
+static constexpr std::uint64_t B2_CYCLIC_DELAY = 1000; ///< 3.5 seconds between slider movements
+static std::uint64_t lastButtonTimestamp = 0;
+static std::uint64_t lastSliderTimestamp = 0;
+
+static bool buttonPressed = false;
+static std::uint16_t buttonTransitions = 0;
+
+static constexpr std::uint16_t SLIDER_MAX_POSITION = 0xFAFF;
+static bool backToZero = false;
+static std::uint16_t sliderPosition = 0;
+static std::atomic_bool running = { true };
+
+using namespace std;
+
+void signal_handler(int)
+{
+	running = false;
+}
+
+void simulate_button_press()
+{
+	buttonPressed = !buttonPressed;
+	TestVirtualTerminalClient->update_auxiliary_input(AUXN_INPUT_BUTTON, buttonPressed, buttonTransitions);
+	// std::cout << "Auxiliary Button input value change send (pressed: " << buttonPressed << ", transitions: " << buttonTransitions << ")" << std::endl;
+	buttonTransitions++;
+}
+
+void simulate_slider_move()
+{
+	constexpr std::uint16_t AMOUNT_TO_MOVE = SLIDER_MAX_POSITION / 20; // We should now be able to reach the end of the slider in 20 steps
+	if (backToZero)
+	{
+		if (sliderPosition > AMOUNT_TO_MOVE)
+		{
+			sliderPosition -= AMOUNT_TO_MOVE;
+		}
+		else
+		{
+			sliderPosition = 0;
+			backToZero = false;
+		}
+	}
+	else
+	{
+		if (sliderPosition < SLIDER_MAX_POSITION - AMOUNT_TO_MOVE)
+		{
+			sliderPosition += AMOUNT_TO_MOVE;
+		}
+		else
+		{
+			sliderPosition = SLIDER_MAX_POSITION;
+			backToZero = true;
+		}
+	}
+	TestVirtualTerminalClient->update_auxiliary_input(AUXN_INPUT_SLIDER, sliderPosition, 0xFFFF);
+	// std::cout << "Auxiliary slider input value change send (value1: " << sliderPosition << ")" << std::endl;
+}
+
+void update_CAN_network()
+{
+	if (nullptr != TestVirtualTerminalClient &&
+	    !TestVirtualTerminalClient->get_auxiliary_input_learn_mode_enabled())
+	{
+		if (isobus::SystemTiming::time_expired_ms(lastButtonTimestamp, BUTTON_CYCLIC_DELAY))
+		{
+			lastButtonTimestamp = isobus::SystemTiming::get_timestamp_ms();
+			simulate_button_press();
+		}
+		if (isobus::SystemTiming::time_expired_ms(lastSliderTimestamp, B2_CYCLIC_DELAY))
+		{
+			lastSliderTimestamp = isobus::SystemTiming::get_timestamp_ms();
+			simulate_slider_move();
+		}
+	}
+
+	isobus::CANNetworkManager::CANNetwork.update();
+}
+
+void raw_can_glue(isobus::HardwareInterfaceCANFrame &rawFrame, void *parentPointer)
+{
+	isobus::CANNetworkManager::CANNetwork.can_lib_process_rx_message(rawFrame, parentPointer);
+}
+
+// This callback will provide us with event driven notifications of auxiliary input from the stack
+void handle_aux_input(isobus::VirtualTerminalClient::AssignedAuxiliaryFunction function, std::uint16_t value1, std::uint16_t value2, isobus::VirtualTerminalClient *)
+{
+	std::cout << "Auxiliary function event received: (" << function.functionObjectID << ", " << function.inputObjectID << ", " << static_cast<int>(function.functionType) << "), value1: " << value1 << ", value2: " << value2 << std::endl;
```

Comment:
Mostly a nit, if you are `using namespace std` I'd suggest either omitting `std::` everywhere or removing `using namespace std`

Reply:
Removed `using namespace std` from all examples as we never omitted `std::` by the looks of it

Reply:
Actually nevermind, for some reason my windows machine decided building was fine while it clearly isn't

Reply:
`std::` was only not present for some of the `cout` and `endl` in the transport_layer example, so I added it instead of adding back the `using namespace std`. Though I'll change it back happily if the opposite is desired